examples/chef-server/README.md

```diff
@@ -5,6 +5,26 @@ This directory contains a simple vagrant setup that expects you have a Chef Serv
 1.) Upload cookbook to Chef Server
 
 ```
```

Comment:
You may just want a comment like "From within the cookbook you can run the following commands:" type of thing so they know `berks` is run from command prompt inside the cookbook directory.